### PR TITLE
Japanese translation.

### PR DESCRIPTION
**Pull request by momo-i**

### CODE REVIEW

File: language/ja/info_acp_recenttopics.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ *
+ * @package Recent Topics Extension
+ * English translation by PayBas
```
Comment (PayBas) [inline language/ja/info_acp_recenttopics.php]: Please provide your name ;)

File: language/ja/info_acp_recenttopics.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	'RT_MIN_TOPIC_LEVEL'			=> 'トピックタイプの最大レベル',
+	'RT_MIN_TOPIC_LEVEL_EXP'		=> '表示するトピックタイプの最大レベルを決定します。それだけで設定されたレベル以上のトピックが表示されます。<br />(0 = 通常, 1 = 注目, 2 = 告知, 3 = 全体告知)',
+	'RT_NUMBER'						=> '新着トピック数',
+	'RT_NUMBER_EXP'					=> '表示するトピック数です。',
+	'RT_PAGE_NUMBER'				=> '新着トピックページ数',
+	'RT_PAGE_NUMBER_EXP'			=> 'ページネーションを使用して多くの新着トピックを表示できます。この機能を無効にするには 1 を入力します。0 を入力した場合、掲示板の全てのトピックを表示するために必要なだけのページが存在するでしょう。(お勧めしません)',
+	'RT_PARENTS'					=> '上位フォーラムを表示',
+	'RT_PARENTS_EXP'				=> '新着トピック行の中に上位フォーラムを表示します。',
+	'RT_SORT_START_TIME'			=> 'トピック開始時間でソート',
+	'RT_SORT_START_TIME_EXP'		=> 'Enable to sort recent topics by the starting time of the topic, instead of the last post time.',
```
Comment (PayBas) [inline language/ja/info_acp_recenttopics.php]: Missing?

File: language/ja/recenttopics.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ *
+ * @package Recent Topics Extension
+ * English translation by PayBas
```
Comment (PayBas) [inline language/ja/recenttopics.php]: Please provide your name ;)

**Comment by momo-i** [comment]
Sorry, translated 1 word and changed english to japanese in comment.

**Pull request merged**